src/physics.js

```
@@ -58,26 +74,158 @@ export function setupPhysics(aircraft, onTakeoff, terrain, ocean) {
+    }
     const drag = aircraft.velocity.clone().multiplyScalar(
-      -0.5 * AIR_DENSITY * speed * DRAG_COEFF * WING_AREA / MASS
+      -0.5 * AIR_DENSITY * speed * dragCoeff * WING_AREA / MASS
     );
+    
     const liftMag = 0.5 * AIR_DENSITY * speed * speed * WING_AREA * LIFT_COEFF;
-    const lift = up.clone().multiplyScalar((aircraft.position.y > 5 ? liftMag : 0) / MASS);
+    const baseDrag = 0.5 * AIR_DENSITY * speed * dragCoeff * WING_AREA / MASS;
+    // Add progressive drag that increases with speed for more realistic acceleration curve
+    const speedFactor = 1 + (speed / MAX_SPEED) * 0.8; // Progressive drag increase
+    const drag = aircraft.velocity.clone().multiplyScalar(-baseDrag * speedFactor);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'drag'. Consider to delete it or rename it.

'drag' is defined here:


(lint/suspicious/noRedeclare)